### PR TITLE
Refactor: Simplify code in scripts

### PR DESCRIPTION
**Pull request by eumiro**

### CODE REVIEW

File: scripts/ci/runners/sync_authors.py [hunk 1]
```diff
@@ -42,15 +42,8 @@
...
-for membership in author_list:
-    author_set.update([author for author in author_list[membership]])
-
-authors = ""
-for author in sorted(author_set):
-    authors += f'            "{author}",\n'
-
-authors = authors[:-2]
+author_set = set().union(*author_list.values())
+authors = ",\n".join(f'            "{author}"' for author in author_set)
```
Comment (uranusjr) [inline scripts/ci/runners/sync_authors.py]:
```suggestion
authors = ",\n".join(f'            "{author}"' for author in sorted(set(author_list.values())))
```

Comment (eumiro) [reply scripts/ci/runners/sync_authors.py]: Looks like https://github.com/apache/airflow-ci-infra/pull/58 removed that authors file anyway.

Comment (hussein-awala) [reply scripts/ci/runners/sync_authors.py]: Indeed, I opened #33350 to remove the file, could you revert this change?

**Review by hussein-awala — APPROVED** [review]
LGTM after reverting the changes on scripts/ci/runners/sync_authors.py

**Comment by eumiro** [comment]
@hussein-awala reverted, rebased.

**Review by potiuk — APPROVED** [review]

**Pull request merged**